azure-pipelines/secure-development-tools.yml

```diff
@@ -6,16 +6,6 @@ steps:
 - powershell: echo "##vso[build.addbuildtag]compliance"
   displayName: 🏷️ Tag run with 'compliance'
 
-- task: CredScan@3
-  displayName: 🔍 Run CredScan
-
-- task: PoliCheck@2
-  displayName: 🔍 Run PoliCheck
-  inputs:
-    targetType: F
-    targetArgument: $(System.DefaultWorkingDirectory)
-    optionsUEPATH: $(System.DefaultWorkingDirectory)\azure-pipelines\PoliCheckExclusions.xml
-
 - task: CopyFiles@2
   displayName: 📂 Collect APIScan/BinSkim inputs
```

Comment:
I guess we should revise this to not mention BinSkim anymore, since you removed the binskim task.